docs-go/cloud-run.md

```diff
@@ -139,14 +140,14 @@ flow.
     - {Gemini (Google AI)}
 
       ```posix-terminal
-      gcloud run deploy --port 3400 \
+      gcloud run deploy \
```

Comment:
Any reason you removed this? I'm not sure whether it's needed or not, I just don't think anything has changed in terms of specifying a port.

Reply:
I thought it's preferable to just use the port that cloud run sets via PORT when you deploy